src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -238,7 +251,8 @@
              (every? #(isa? % Number) [(fst json-column) (snd json-column)])
              [json-column java.lang.Number]
 
-             (every? #{java.lang.String java.lang.Long java.lang.Integer java.lang.Double java.lang.Boolean}
+             (every? #{java.lang.String java.lang.Long java.lang.Integer
+                       java.lang.Double java.lang.Boolean java.time.LocalDateTime}
                      [(fst json-column) (snd json-column)])
```

Comment:
I know this code isn't part of this PR per se but `(fst json-column)` and `(snd json-column)` is a confusing to me not knowing what this code is or is supposed to do. I'm having to carefully read the function to try to wrap my head around it so I can make meaningful comments. This is what I've been able to gather:

- `json-column` is a keyword, right? 
- `fst` and `snd` are maps?
- `fst` is short for _first_ and `snd` is short for _second_... first and second _what_ I have no idea. 
- `fst` is the accumulator since this is a reducing function. 
- This whole thing returns a map of column keyword -> Class

Besides that I still don't understand what we're doing. If you change `fst` and `snd` to names that are a little more descriptive like `first-field-type-map` or just `field->type` or whatever (not sure since I don't 100% understand what's happening) that would go a long way towards making this code easier to read and thus easier to review